static/app/components/platformPicker.tsx

```diff
@@ -23,6 +27,8 @@ export const PLATFORM_CATEGORIES: {
   platforms?: PlatformKey[];
 }[] = [...JSON.parse(JSON.stringify(categoryList)), {id: 'all', name: t('All')}];
 
+const activePlatforms = platforms.filter(({id}) => !deprecatedPlatforms.has(id));
+
```

Comment:
I am only filtering it out in this component so no functionality breaks for existing projects.
E.g. someone could still try to open the getting-started docs for this platform.